### PR TITLE
buku: fixes for Linuxbrew

### PR DESCRIPTION
**Pull request by maxim-belkin**

### CODE REVIEW

**Comment by maxim-belkin** [comment]
The only thing that fails is test but I think it fails because of problems with `brew`, not the formula

File: Formula/buku.rb [hunk 1]
```diff
@@ -88,6 +96,11 @@ def install
   test do
     ENV["LC_ALL"] = "en_US.UTF-8"
     ENV["XDG_DATA_HOME"] = "#{testpath}/.local/share"
+    unless OS.mac?
+      ENV["PYTHONIOENCODING"] = "utf-8"
+      ENV.append "CFLAGS", "-I#{Formula["libffi"].opt_include}"
```
Comment (rwhogg) [inline Formula/buku.rb]: This is kinda weird. At least on the bottled version, `libffi`'s include directory is in a really weird place:

```
brew list libffi
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/INSTALL_RECEIPT.json
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/README
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/.brew/libffi.rb
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/LICENSE
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/ChangeLog
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/lib/libffi-3.2.1/include/ffi.h
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/lib/libffi-3.2.1/include/ffitarget.h
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/lib/pkgconfig/libffi.pc
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/lib/libffi.so
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/lib/libffi.a
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/lib/libffi.so.6.0.4
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/lib/libffi.so.6
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/share/info/libffi.info
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/share/man/man3/ffi_call.3
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/share/man/man3/ffi_prep_cif_var.3
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/share/man/man3/ffi.3
/home/linuxbrew/.linuxbrew/Cellar/libffi/3.2.1/share/man/man3/ffi_prep_cif.3
```

However, `opt_include` doesn't point to that location, it points to `#<Pathname:/home/linuxbrew/.linuxbrew/opt/libffi/include>`.

So part of me is wondering if `libffi` is supposed to be like that or if the formula is installing it incorrectly.

Comment (maxim-belkin) [inline Formula/buku.rb]: I vaguely recall stumbling upon this (weird) `include` directory (though, I might be confusing it with some other formula). @iMichka added this piece, and I don't know the purpose of it (there is no comment associated with this change). So let's see if we need this at all.

Comment (maxim-belkin) [inline Formula/buku.rb]: It turns out we don't need this piece to install `buku` at all. However, if I don't have `utf-8` thing set, I get an error when I run `buku` in my container:

```
Traceback (most recent call last):
  File "/home/linuxbrew/.linuxbrew/bin/buku", line 3317, in <module>
    main()
  File "/home/linuxbrew/.linuxbrew/bin/buku", line 2945, in main
    argparser.print_help(sys.stdout)
  File "/home/linuxbrew/.linuxbrew/bin/buku", line 1854, in print_help
    self.program_info(file)
  File "/home/linuxbrew/.linuxbrew/bin/buku", line 1827, in program_info
    ''' % (__version__, __author__, __license__))
UnicodeEncodeError: 'ascii' codec can't encode character '\xa9' in position 131: ordinal not in range(128)
```
And, by the way, tests still fail:

```
=> /home/linuxbrew/.linuxbrew/Cellar/buku/3.1/bin/buku --np --sany Homebrew
Error: uninitialized constant Homebrew::MiniTest
/home/linuxbrew/.linuxbrew/Library/Homebrew/dev-cmd/test.rb:86:in `rescue in block in test'
/home/linuxbrew/.linuxbrew/Library/Homebrew/dev-cmd/test.rb:94:in `block in test'
/home/linuxbrew/.linuxbrew/Library/Homebrew/dev-cmd/test.rb:29:in `each'
/home/linuxbrew/.linuxbrew/Library/Homebrew/dev-cmd/test.rb:29:in `test'
/home/linuxbrew/.linuxbrew/Library/Homebrew/brew.rb:93:in `<main>'
```

Comment (rwhogg) [inline Formula/buku.rb]: > It turns out we don't need this piece to install `buku` at all.

Sounds good. I agree with all the other changes.

**Pull request closed without merging**